### DISCUSSION TITLE
ScriptReference/EditorUtility

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#165] [Notes]
# ScriptReference/EditorUtility

https://docs.unity3d.com/ScriptReference/EditorUtility.html

**Comment by vertxxyz** [comment]
| | |
| --- | --- |
|  `RevealInFinder` | Shows a file or folder in the OS' file explorer. The parent folder will be shown with the item selected. |
| `OpenWithDefaultApp` | Opens a file using the default application for the file type.<br />If you pass a folder path to the function it will open it, showing its contents. |
